### DISCUSSION TITLE
客户端添加游戏版本联网更新功能

### DISCUSSION THREAD

**Discussion by Halogly** [Meloong-Git/PCL#3509] [提问] [answered]
如同主页的联网更新一般，希望能添加游戏版本的联网更新下载。

一般服主都会发布自己的客户端给玩家，如果一段时间后需要更改客户端的内容玩家就得从服主那里重新下载客户端或者整合包，不是特别方便。

**Comment by wuliaodexiaoluo** [comment]
> 如同主页的联网更新一般，希望能添加游戏版本的联网更新下载。
> 
> 一般服主都会发布自己的客户端给玩家，如果一段时间后需要更改客户端的内容玩家就得从服主那里重新下载客户端或者整合包，不是特别方便。

已经有程序实现了这个功能
并且实现此功能需要额外做 HTTP 服务端，龙猫 100% 会拒绝，因为这不是他应该关心的问题

**Reply by Halogly** [reply]
服务器我觉得服主自己提供就行，毕竟整合包一般比较大，龙猫肯定负担不起，所以我的意思是能加这个功能，但是下载更新的链接应该由服主自己提供

**Reply by Halogly** [reply]
话说有什么程序？我不知道

**Reply by MoYuan-CN** [reply]
类似 HMCL 的打包整合包时配置更新地址所打包出来的吗

**Reply by ghost** [reply]
不不不，你误解了我的意思，我指的是需要一个基于http且支持ssl的更新服务端
自动更新的软件你可以加这个群：925057600

**Reply by Halogly** [reply]
人好像不一样？不管了...基于http和ssl的服务端我也有啊，我是指腐竹自己提供，不过不行的话我去群上看看好了，谢谢

**Reply by wuliaodexiaoluo** [reply]
不可能都请求一遍吧，json也要写。。。

**Reply by Halogly** [reply]
请求什么🤔

**Reply by wuliaodexiaoluo** [reply]
文件，不能每次启动都历遍请求文件

**Comment by MoYuan-CN** [comment, marked as answer]
见  #1768 #3317
> 由于目前 MCBBS v2 整合包规范缺乏开发文档，在他们那边打算补上文档之前都没法继续制作该规范的额外特性。

**Comment by Halogly** [comment]
好的
________________________________
发件人: 凌云 ***@***.***>
发送时间: 2024年3月11日 17:42
收件人: Hex-Dragon/PCL2 ***@***.***>
抄送: HoneyAlloy ***@***.***>; Author ***@***.***>
主题: Re: [Hex-Dragon/PCL2] 客户端添加游戏版本联网更新功能 (Discussion #3509)


见 #1768<https://github.com/Hex-Dragon/PCL2/issues/1768> #3317<https://github.com/Hex-Dragon/PCL2/issues/3317>

―
